### ISSUE TITLE
My avatar's special gear does not appear on WindowsPhone

### ISSUE THREAD

**Issue by dsnunez** [negue/HabitRPG-Metro#13]
On the Habitica web my avatar looks like this:
![avatarweb](https://cloud.githubusercontent.com/assets/2080810/11995810/3898fc74-aa36-11e5-9a88-684757105510.PNG)
(It's wearing the Snowboard mage set of the Limited Editon 2015-2016 Winter Gear)

But on my windows phone, my avatar looks like this:
![wp_ss_20151224_0001](https://cloud.githubusercontent.com/assets/2080810/11995860/e57dce2e-aa36-11e5-83a5-f9456f24be17.png)

**Comment by negue** [comment]
yeah it's been a while since I updated my app, I'll fix that asap (I'll also use the same technique as the other native apps do) I'll load those images one by one instead of having all of them in my app package.

I'll write here again once I  have a new version
